downloads.markdown

```diff
@@ -6,48 +6,40 @@ isDownloads: true
 
 # Downloads
 
-There are three widely used ways to install the Haskell toolchain on supported platforms. These are:
-
-*   [Minimal installers](#minimal): Just GHC (the compiler), and build tools (primarily Cabal and Stack) are installed globally on your system, using your system's package manager.
-
-*   [Stack](#stack): Installs the `stack` command globally: a project-centric build tool to automatically download and manage Haskell dependencies on a project-by-project basis.
-
-*   [Haskell Platform](#platform): Installs GHC, Cabal, and some other tools, along with a starter set of libraries in a global location on your system.
+## Haskell Platform
```

Comment:
There exists no haskell platform anymore.

For linux and mac, it redirects to either 1. package managers or 2. ghcup. This is another layer of indirection causing confusion for users.

For windows it redirects to chocolatey. And windows is an even more complicated matter being discussed: https://gitlab.haskell.org/ghc/ghc/issues/18104

All this information should be **right here**, not on the haskell platform page.

Reply:
I disagree. We should point to the raw binaries, and then just point to the platform page for the recommended installation methods for each system.

Reply:
The section can be simplified by saying that the platform points to recommended installers for common systems, and leave it at that.

Reply:
@gbaz I do not understand your remarks. There are no binaries on the haskell platform page. The term "haskell platform" by now is quite confusing, because it doesn't exist. It just redirects the user further.

Reply:
What I'm suggesting is that we can point to ghc's raw binary downloads somewhere on this page, so people are aware of them. Independently, we can also point to the platform and all we need say about it is it collects recommended installers for common systems.

I am suggesting that extra hop is _fine_, since the platform "brand" can then stay uniform across different systems. It is just that the extra hop can now be significantly simplifed, to make this section very short and clear.

"Raw binaries of ghc are available from [..]. Most people will want to use an installation system to manage ghc and related tools such as cabal. The haskell platform collects recommended installation methods across common platforms [...]." Done.

Reply:
> What I'm suggesting is that we can point to ghc's raw binary downloads somewhere on this page, so people are aware of them. Independently, we can also point to the platform and all we need say about it is it collects recommended installers for common systems.

If you are referring to the bindists, then I'm pretty sure most novice users will not be able to install these. That is basically one of the problems ghcup tried to resolve and it's still getting lots of bug reports from mac users, who don't know how to adjust their PATH variable. Download, unpacking, dealing with configure flags, custom locations, build requirements, ...is **advanced** use. If it should be mentioned on the landing page, then in a way that makes it clear for users this isn't something they want generally, unless they know what they are doing.

Otherwise we'll have to deal with more confused users in `#haskell`, I am pretty sure.

> I am suggesting that extra hop is fine, since the platform "brand" can then stay uniform across different systems. It is just that the extra hop can now be significantly simplifed, to make this section very short and clear.

And I am strongly against this. The entire reason of the issue I raised at #12 was that this extra hop is confusing and the branding "haskell platform" confuses users further.

You're landing on the download page and the download page tells you "To get to the download page, click here, but we call it 'haskell platform', because of legacy reasons, when it was still an installer."

Reply:
What should I do? Remain Installer in place or remove it?

My suggestion is even more bold. Integrate Haskell-Platform site to downloads and create 'Advanced' section with the other options. How's that?

Reply:
More proof that haskell platform both as branding and subpage is confusing users: https://www.reddit.com/r/haskell/comments/guecax/confused_by_the_haskell_installation_options

@mattbaumann my opinion is to remove all mentions of "haskell platform" as well as stack and provide a concise list of options per platform. Whether that follows the design of current haskell platform or not doesn't matter much and should be up to whomever provides that PR.